network/hostnetwork/dht.go

```diff
@@ -402,7 +400,7 @@ func (dht *DHT) iterateBootstrapHosts(
 			if err != nil {
 				continue
 			}
-			log.Println("sending ping to " + bn.Address.String())
+			log.Debugln("sending ping to " + bn.Address.String())
```

Comment:
Almost everywhere use write to log only warnings and error, and as a result you don't have kind of "access log" ( you have it only in debug mode. ). But normally, system will work without debug mode, and it will be hard to understand without "access log".
One more thing - I think you log something like "query id". It's very helpful thing

Reply:
Default log level is Info, debug messages added for debugging and may be will removed in the future

Reply:
Yes, I mean, that you don't have information about system until everything is wrong

Reply:
we also have metrics counters which shows that system is working